docs/contributing/continuing-unfinished-work.md

```diff
@@ -0,0 +1,83 @@
+# Continuing unfinished work
+
+Sometimes, work on an issue is started, but not brought to completion. This may
+happen for a variety of reasons — the contributor working on the project gets
+busy, maintainers cannot prioritize reviewing the work, a contributor doesn't
+have the skills required to complete the project, there is
+an unexpected technical challenge or blocker, etc.
+
+Completing work that someone else has started is a great way to contribute! Here
+are the steps required:
+
+1. [Find work to be completed.](#find-work-to-be-completed)
+1. [Review existing work and feedback.](#review-existing-work-and-feedback)
+1. [Decide how to use prior work.](#decide-how-to-use-prior-work)
+1. [Present your pull request.](#present-your-pull-request)
+
+## Find work to be completed
+
+In Zulip's server and web app [repository](https://github.com/zulip/zulip), pull
+requests that have significant work towards something valuable are often tagged
+with a [completion candidate label][completion-candidate] label. You can review
+this label for issues that you find interesting and have the skills to complete.
+
+In general, it's common to see one or more pull request linked to an issue
+you're interested in.
```

Comment:
I'm not sure what point this is making.

Reply:
That all these guidelines apply in general, not only when you set out to find a PR to finish. I'll try making it more explicit.